backend/src/main/resources/liquibase/create-tables-changelog-1.0.xml

```diff
@@ -1238,4 +1238,48 @@
                                  referencedColumnNames="id"
                                  referencedTableName="keyvalue"/>
     </changeSet>
+
+    <changeSet author="lineage" id="new_lineage">
```

Comment:
keep each table in a different changeset.
add a precondition of not exists

Reply:
I have create separate changesets for it.
To add this precondition I should just add a changeset or add some parameter?